nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -37,10 +37,9 @@ let
       . /etc/profile
       cd "$HOME"
 
-      # Allow the user to execute commands at the beginning of the X session.
-      if test -f ~/.xprofile; then
-          source ~/.xprofile
-      fi
+      # Source profiles similar to https://github.com/sddm/sddm/blob/develop/data/scripts/Xsession
+      [ -f .profile ] && . .profile
+      [ -f .xprofile ] && . .xprofile
```

Comment:
This is equivalent test call that was used before, but I think we want to keep the test call for consistency with the rest of the file.

Also I think it might be good to group sourcing `~.profile` with sourcing `/etc/profile` because those are the two pieces that emulate the environment that would be setup by a bash login shell. `.xprofile` is it's own thing.

Reply:
I moved it and used `test` like the rest of the file. I don't know what `ofborg-eval` is but it's failing and I can't figure out why. The code works and the other checks are passing. Is it required to pass for a merge?

Reply:
Yeah it doesn't really make much sense, I tried to re-run it.